portal-api/api-tests/v1/users/login-with-sign-in-link.api-test.js

```diff
@@ -111,26 +121,30 @@ describe('POST /users/:userId/sign-in-link/:signInToken/login', () => {
         expect(status).toBe(403);
         expect(body).toStrictEqual({
           message: 'Forbidden',
-          errors: [{
-            msg: `Invalid sign in token for user ID: ${testUserId}`
-          }]
+          errors: [
+            {
+              msg: `Invalid sign in token for user ID: ${testUserId}`,
+            },
+          ],
         });
       });
     });
 
     describe('when the user has a sign in token saved', () => {
       describe('when the signInToken does not match the saved sign in token', () => {
         beforeEach(async () => {
-          await (await usersCollection()).updateOne(
+          await (
+            await usersCollection()
+          ).updateOne(
             { _id: { $eq: testUser._id } },
             {
               $set: {
                 signInToken: {
                   saltHex: saltHexForValidSignInToken,
                   hashHex: nonMatchingHashHex,
-                  expiry: Date.now() + (30 * 60 * 1000),
-                }
-              }
+                  expiry: Date.now() + 30 * 60 * 1000,
```

Comment:
Can we add a comment stating `30 minutes expiry` please.

Reply:
added as a named variable instead of comment